src/StructuredLogger/Construction/MessageProcessor.cs

```diff
@@ -219,25 +219,21 @@ public void AddMessage(LazyFormattedBuildEventArgs args, string message)
             {
                 node = construction.Build;
 
-                if (message.StartsWith("Overriding target"))
-                {
+                if(message.StartsWith("Overriding target")) {
                     var folder = construction.Build.GetOrCreateNodeWithName<Folder>("TargetOverrides");
                     folder.IsLowRelevance = true;
                     node = folder;
                     messageNode.IsLowRelevance = true;
-                }
-                else if (message.StartsWith("The target") && message.Contains("does not exist in the project, and will be ignored"))
-                {
+                } else if(message.StartsWith("The target") && message.Contains("does not exist in the project, and will be ignored")) {

```

Comment:
Not intentional in the sense of wanting to change all formatting. I altered my settings ages ago; I'll reset an instance of VS to default and fix in the morning.